README.md

```
@@ -1,8 +1,127 @@
+
+### テスト・品質チェック
+
+```bash
+pnpm lint              # コード品質チェック
+pnpm lint:fix          # 自動修正
+pnpm typecheck         # 型チェック
+```
+
+## 🌐 開発 URL
+
+- **フロントエンド**: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)